services/src/pro/datasets/postgres.rs

```diff
@@ -680,23 +691,70 @@ where
     ) -> Result<LayerCollection> {
         let conn = self.conn_pool.get().await?;
 
+        let coll_id = &collection.0;
+
+        if coll_id == "" || coll_id == &DATASET_DB_ROOT_COLLECTION_ID.to_string() {
```

Comment:
we would have to make this root collection accessible in the UI somehow

Reply:
yes the root collection is now the "User Data". However there might be persons who will find that strange...